scripts/run_checks_on_excluded_packages.sh

```diff
@@ -2,29 +2,50 @@
 
 set -e
 
-# `packages` should reflect `exclude` section from root `Cargo.toml`.
-packages=(
-  "flooder"
-  "e2e-tests"
-  "aleph-client"
-  "fork-off"
-  "benches/payout-stakers"
-  "bin/cliain"
-  "contracts/access_control"
-  "contracts/button"
-  "contracts/game_token"
-  "contracts/marketplace"
-  "contracts/simple_dex"
-  "contracts/ticket_token"
-  "contracts/wrapped_azero"
-  "contracts/adder, "
-)
-
-for p in ${packages[@]}
-do
+TOML_FILE="Cargo.toml"
+RUST_TOOLCHAIN=nightly-2022-10-30-x86_64-unknown-linux-gnu
```

Comment:
Maybe we should read the value from `rust-toolchain` file?

Reply:
good idea, added